### PR TITLE
task(auth): Improve usablity of security event records

### PR DESCRIPTION
**Pull request by dschom**
## Because

- We want to be able to see ip values in security events tables
- We have logs with these values
- We do not retain these values indefinitely

## This pull request

- Adds a readable ip to security events

## Issue that this pull request solves

Closes: FXA-9115

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [x] I have added necessary documentation (if appropriate).
- [x] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)
image

## Other information (Optional)

Any other information that is important to this pull request.

### CODE REVIEW

File: packages/fxa-shared/db/models/auth/security-event.ts [hunk 1]
```diff
@@ -87,7 +87,8 @@ export class SecurityEvent extends BaseAuthModel {
         tokenId ? uuidTransformer.to(tokenId) : null,
         EVENT_NAMES[name],
         ipAddrHmac,
-        Date.now()
+        Date.now(),
```
Comment (vbudhram) [inline packages/fxa-shared/db/models/auth/security-event.ts:90]: Do we need to add `ipAddr` in table field name on line 64?

**Review by vbudhram — APPROVED** [review]
@dschom Works as expected 👍🏽

I'm not sure exactly how long we keep these events around but I think we can do a follow up to deprecate `ipAddrHmac`, update `findByUidAndIP` or any other functions to use ip instead.

Also do we want to update the get security event functions? It would make it a little easier to assert values are getting set correctly.

**Comment by vbudhram** [comment]
Also add migration to deploy docs

**Comment by dschom** [comment]
@vbudhram Thanks for the review. I pushed one small update so you can validate these via gql. I'll file some follow up tickets too.

```query getAccount {
  account{
    uid,
    securityEvents{
      name,
      ipAddr
    }
  }
}```

**Review by vbudhram — APPROVED** [review]
:shipit:

**Pull request merged**